File: c/Lista3/Desafio6.c
#include <stdio.h>
#include <stdlib.h>
#include <math.h>

main(){
    
    int A = 10, B = 5;

    float adicao, subtracao, divisao, multiplicacao;
    
printf("Selecione a operação: ");
printf("\n");
printf("\n");

adicao = A + B;
subtracao = A -B;
divisao = A / B;
multiplicacao = A * B;

printf("\n");
printf("A adição de X + Y = %3.0f", adicao);
printf("\n");
printf("A subtração de X - Y = %3.0f", subtracao);
printf("\n");
printf("A divisão de X / Y = %3.0f", divisao);
printf("\n");
printf("A multiplicação de X * Y = %3.0f", multiplicacao);
    
}

File: js/lista3/Desafio6.js
const readline = require('readline');
const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
}); 

/*
Desafio6- Faça um algoritmo que leia dois números inteiros e efetue as operações de adição, subtração, multiplicação e divisão de A por B:

Apresentar ao final os quatro resultados obtidos.
*/

rl.question('Digite o valor de A: ', (a) => {
    rl.question('Digite o valor de B: ', (b) => {

const soma = a + b;
const subtracao = a - b;
const divisao = a / b;
const multiplicacao = a * b;  
    
        console.log(`Soma: ${soma}, Subtração: ${subtracao}, Divisão: ${divisao}, Multiplicação: ${multiplicacao}`);
      rl.close();
    });
  });
    



   
